src/clj/game/cards-upgrades.clj

```diff
@@ -364,6 +364,16 @@
    {:init {:root "HQ"}
     :in-play [:hand-size-modification 2]}
 
+   "Ruhr Valley"
+   {:events {:run {:req (req this-server)
+                   :effect (effect (lose :runner :click 1))
+                   :msg "force the runner to spend an additional [Click]"}
+             :runner-turn-begins {:req (req (> (:click-per-turn runner) 1))
+                               :effect (req (enable-run-on-server state card (second (:zone card))))}
+             :runner-spent-click {:req (req (<= 1 (:click runner)))
+                                  :effect (req (prevent-run-on-server state card (second (:zone card))))}
```

Comment:
For this effect (and also above and below), you can use `(effect (prevent-run-on-server (second (:zone card))))` instead of `(req ...)`. `req` and `effect` do the same thing, except `effect` auto-inserts the parameters `state side` as the first two arguments of any function call you make in the `effect`. It's just a minor elegance thing.